WormsApp/worm.cpp

```diff
@@ -28,8 +26,12 @@ void Worm::move() {
   switch (currDir_)
   {
   case LEFT:
-      (x_>0)?--x_:x_%=board_->getWidth();
```

Comment:
I actually thought about adding some spaces: `x_ > 0 ? --x_ : x_ %= board_->getWidth();`, but with `if-else` is even better I think.